Sources/NetworkProtection/Diagnostics/NetworkProtectionConnectionTester.swift

```diff
@@ -37,6 +37,7 @@ final class NetworkProtectionConnectionTester {
 
     enum TesterError: Error {
         case couldNotFindInterface(named: String)
+        case connectionTestFailed
```

Comment:
When starting the VPN on demand, we now do an immediate test on startup.  This is the error if the test fails.